yogstation/code/modules/donor/donor_verbs.dm

```diff
@@ -0,0 +1,143 @@
+
+var/list/donor_verbs_list = list(
+	/client/proc/donor_ooc
+	)
+
+/client/proc/add_donor_verbs()
+	if(is_donator(src))
+		verbs += donor_verbs_list
+
+/client/proc/remove_donor_verbs()
+	if(is_donator(src))
+		verbs.Remove(
+			donor_verbs_list,
+			)
+
+var/global/normal_donor_ooc_colour = "#333333"
+
+/client/proc/donor_ooc_admin(msg as text)
+	set name = "Donor OOC"
+	set category = "Admin"
+	donor_ooc(msg)
+
+/client/proc/donor_ooc(msg as text)
+	set name = "Donor OOC"
+	set category = "OOC"
+
+	if(GLOB.say_disabled)	//This is here to try to identify lag problems
+		to_chat(usr, "<span class='danger'>Speech is currently admin-disabled.</span>")
+		return
+
+	if(!mob)	return
+	if(IsGuestKey(key))
+		to_chat(src, "Guests may not use Donor.")
+		return
+
+	msg = copytext(sanitize(msg), 1, MAX_MESSAGE_LEN)
+	if(!msg)	return
```

Comment:
Don't use this same-line return, I wouldn't be surprised if BYOND didn't actually return because muh indentation.